### PR TITLE
[SW-2703] Add GLM tutorial and expose coefficients

### PR DESCRIPTION
**Pull request by krasinski**

### CODE REVIEW

File: ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala [hunk 1]
```diff
@@ -644,6 +644,35 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
...
+  test("getCoefficients is exposed for trained GLM model") {
+    val model = new H2OGLM()
+      .setSeed(1)
+      .setFeaturesCols("CAPSULE", "RACE", "DPROS", "DCAPS", "PSA", "VOL", "GLEASON")
+      .setLabelCol("AGE")
+      .setLambdaValue(Array(0))
+      .setComputePValues(true)
+      .fit(prostateDataFrame)
+
+    model.getCoefficients().show()
```
Comment (kanech) [inline ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala:656]: This line seems like a left over from exploration and can be probably deleted for an automatic test suite .

**Review by kanech — APPROVED** [review]
Nice work.

File: doc/src/site/sphinx/ml/sw_glm.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: scala
+
+            import org.apache.spark.SparkFiles
+            val datasetUrl = "https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv"
+            spark.sparkContext.addFile(datasetUrl) //for example purposes, on a real cluster it's better to load directly from distributed storage
+            val rawSparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
+            val sparkDF = rawSparkDF.withColumn("CAPSULE", $"CAPSULE" cast "string")
```
Comment (mn-mikke) [inline doc/src/site/sphinx/ml/sw_glm.rst:43]: just FYI: you can also define the conversion by calling the `setColumnsToCategorical` method on the algorithm instance.

**Review by mn-mikke — APPROVED** [review]
LGTM, great job!

**Pull request merged**